panopticum/management/commands/importer.py

```diff
@@ -0,0 +1,1085 @@
+import csv
+import logging
+import re
+import sys
+import json
+import traceback
+
+from django.core.exceptions import MultipleObjectsReturned
+from django.core.management.base import BaseCommand, CommandError
+from django.core import serializers
+
+from panopticum.models import *
+
+
+# Provides all the utility functions either in static form or as part of instance methods.
+class Utility:
+    comp_categories = \
+        {'Category1': {'description': '', 'order': 1}}
+
+    comp_subcategories = \
+        [{'name': 'SubCategory_A', 'description': '', 'category': 'Category1'},
+         {'name': 'SubCategory_B', 'description': '', 'category': 'Category1'},
+         {'name': 'SubCategory_C', 'description': '', 'category': 'Category2'},
+         {'name': 'SubCategory_B', 'description': '',
+          'category': 'Category2'}]  # We can have duplicate SubCategory name
+
+    comp_dataprivacies = \
+        {'Framework': {'order': 1},
+         'Infrastructure': {'order': 2}}  # Order can be same across records.
+
+    comp_runtimes = {"Library", "Framework", "Driver", "OS Service", "Component"}
+
+    loggers = {'Logf'}
+
+    countries = \
+        {'Russia', 'United Kingdom', 'Japan', 'Singapore', 'Germany', 'Estonia'}

```

Comment:
Why we have such a strange set of countries?

Reply:
These were already part of one of the test DB instances, I carry forward. But we can define these with [country codes](https://en.wikipedia.org/wiki/ISO_3166) and country name.
It looks like we don't use the countries table, so we can drop this part altogether.